spec/features/admin/index_spec.rb

```diff
@@ -8,12 +8,13 @@
       expect(page).to have_content("Admin Dashboard")
     end
     
-    it "should display links to admin/merchants and admin/invoices admin/dashboard" do
+    it "should display links to admin/merchants, admin/invoices, and admin/dashboard" do
       visit admin_path
-      save_and_open_page
+      
```

Comment:
Way to get that save_and_open out of there - maybe we should leave on in just for fun though 😆